Extension/src/common.ts

```diff
@@ -218,11 +221,11 @@ export function setProgress(progress: number): void {
         let telemetryProperties: { [key: string]: string } = {};
         let progressName: string;
         switch (progress) {
-            case 0: progressName = "install started"; break;
-            case progressInstallSuccess: progressName = "install succeeded"; break;
-            case progressExecutableStarted: progressName = "executable started"; break;
-            case progressExecutableSuccess: progressName = "executable succeeded"; break;
-            case progressParseRootSuccess: progressName = "parse root succeeded"; break;
+            case 0: progressName = localize("install.started", "install started"); break;
```

Comment:
> Are there any strings in the Debugger code that we need to add to the loc files?

(Adding these in next push)

> Should we pre-loc all strings in each file into const variables so that we don't have to repeatedly look them up? (maybe only for code paths that are exercised multiple times)

If the concern is about efficiency, I don't believe that would be an issue.  vscode-nls-dev does a source rewrite step which converts localize calls to indexed table lookups, so they should be fast.

> Were you going to include the xlf or i18n files as part of this commit?  I think it would be good for community members to be able to submit pull requests against translations if they think they could be improved.

I plan to include the i18n files.  Once these changes are close to done, I will export a file for loc, and include what I get back from them in this PR.  I may wait a while to submit this PR until we've dealt with the native side, as we've discuss potentially routing them via the TS side.